src/party.cpp

```diff
@@ -432,8 +434,9 @@ bool Party::canUseSharedExperience(const Player* player) const
 		}
 	}
 
-	uint32_t minLevel = static_cast<int32_t>(std::ceil((static_cast<float>(highestLevel) * 2) / 3));
```

Comment:
That is because we are comparing it to ``player->getLevel()``, so I think it serves that purpose (i.e. we should know that ``player->getLevel()`` returns a ``uint32_t`` inferring from ``minLevel``'s type) 

Should I change it to ``auto`` nonetheless?